templates/layout/page.html.twig

```diff
@@ -0,0 +1,168 @@
+ * - logged_in: A flag indicating if the user is registered and signed in.
+ * - is_admin: A flag indicating if the user has permission to access
+ *   administration pages.
+ *
+ * Site identity:
+ * - front_page: The URL of the front page. Use this instead of base_path when
+ *   linking to the front page. This includes the language domain or prefix.
+ *
+ * Page content (in order of occurrence in the default page.html.twig):
+ * - messages: Status and error messages. Should be displayed prominently.
+ * - node: Fully loaded node, if there is an automatically-loaded node
+ *   associated with the page and the node ID is the second argument in the
+ *   page's path (e.g. node/12345 and node/12345/revisions, but not
+ *   comment/reply/12345).
+ *
+ * Regions:
+ * - page.header: Items for the header region.
+ * - page.search: Items for site search region.
+ * - page.primary_menu: Items for the primary menu region.
+ * - page.slideshow: Items for the slideshow region on the front page.
+ * - page.front_welcome: Items for the front page welcome region.
+ * - page.top_first: Items for the first top region.
+ * - page.top_second: Items for the second top region.
+ * - page.top_third: Items for the third top region.
+ * - page.breadcrumb: Items for the breadcrumb region.
+ * - page.content: The main content of the current page.
+ * - page.sidebar_first: Items for the first sidebar.
+ * - page.sidebar_second: Items for the second sidebar.
+ * - page.bottom_widget_1: Items for the first bottom widget region on the front page.
+ * - page.bottom_widget_2: Items for the second bottom widget region on the front page.
+ * - page.bottom_widget_3: Items for the third bottom widget region on the front page.
+ * - page.footer_first: Items for the first footer region.
+ * - page.footer_second: Items for the second footer region.
+ * - page.footer_third: Items for the third footer region.
+ * - page.footer_fourth: Items for the fourth footer region.
+ * - page.footer: Items for the bottom footer region.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ */
+#}
+<div id="header_wrapper">
+  <div id="inner_header_wrapper">
+    <header id="header">
+      <div class="top_left">
+        {{ page.header }}
+      </div>
+      <div id="headerHero" class="top_right">
+        {{ page.search }}
+        <button class="burger-toggle collapse" type="button" data-toggle="collapse" data-target=".main-menu">
+          <span class="sr-only">Menu</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      </div>
+      <div class="clear"></div>
+    </header>
+  </div>
+  {% if page.primary_menu %}
+    {{ page.primary_menu }}
+  {% endif %}
+</div>
+
+{% if is_front %}
+  {{ page.slideshow }}
+  {{ page.front_welcome }}
+
+  {% if page.top_first or page.top_second or page.top_third %}
+    <div id="top-area" class="clearfix">
+      <div class="column">
```

Comment:
conditionally render the `div.column` only if there is content for the subsequent region which it is wrapping, ie.
`{% if page.top_first %}
    <div class="column">
      {{ page.top_first }}
    </div>
{% endif %}`